dashboards-observability/public/components/application_analytics/home.tsx

```diff
@@ -215,7 +221,12 @@ export const Home = (props: HomeProps) => {
 
   // Rename an existing application
   const renameApp = (newAppName: string, appId: string) => {
-    if (!isNameValid(newAppName)) {
+    if (
+      !isNameValid(
+        newAppName,
+        applicationList.map((obj) => obj.name)
+      )
+    ) {
       setToast('Invalid Application name', 'danger');
```

Comment:
ideally this message should come from `isNameValid` so user know why it's invalid

Reply:
I'll add this in the next PR!